packages/components/psammead-brand/src/index.jsx

```diff
@@ -19,6 +22,11 @@ const PADDING_AROUND_SVG_BELOW_600PX = 32;
 const conditionallyRenderHeight = (svgHeight, padding) =>
   svgHeight ? `height: ${(svgHeight + padding) / 16}rem` : '';
 
+const SvgWrapper = styled.div`
+  max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX};
```

Comment:
Even though [they are the same value](https://github.com/bbc/psammead/blob/7cfe6c0535ed6f95e98ee4fd7eeefc55b212e714/packages/utilities/gel-foundations/src/breakpoints.js#L29-L31) 🙈 Can you use `GEL_GROUP_5_SCREEN_WIDTH_MIN` here instead please :P Its what we used before :)